### PR TITLE
Add temporary employee frontend

### PR DESCRIPTION
**Pull request by patari**
#### Summary

Frontend for https://github.com/espoon-voltti/evaka/pull/3477

### CODE REVIEW

File: frontend/src/employee-frontend/components/unit/tab-unit-information/DaycareAclAdditionModal.tsx [hunk 1]
```diff
@@ -64,12 +76,27 @@ export default React.memo(function DaycareAclAdditionModal({
...
+    lastName: '',
     selectedEmployee: null,
     selectedGroups: null,
-    hasStaffOccupancyEffect: null
+    hasStaffOccupancyEffect: null,
+    pinCode: ''
   })
 
   const submit = useCallback(async () => {
+    if (formData.type === 'TEMPORARY') {
```
Comment (reynders) [inline frontend/src/employee-frontend/components/unit/tab-unit-information/DaycareAclAdditionModal.tsx:89]: First selecting a normal employee and then creating and sending temporary employee sends the normal employee in the POST payload

File: frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx [hunk 2]
```diff
@@ -21,6 +21,24 @@ import { isPinCodeLocked, updatePinCode } from '../../api/employees'
 import { useTranslation } from '../../state/i18n'
 import { useWarnOnUnsavedChanges } from '../../utils/useWarnOnUnsavedChanges'
 
+const badPins = [
```
Comment (reynders) [inline frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx:24]: Setting a bad PIN disables the send button but does not explain why it is disabled

**Review by reynders — CHANGES_REQUESTED** [review]

File: frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx (hunk 2 above)
Comment (patari) [reply frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx:24]: Fixed!

File: frontend/src/employee-frontend/components/unit/tab-unit-information/DaycareAclAdditionModal.tsx (hunk 1 above)
Comment (patari) [reply frontend/src/employee-frontend/components/unit/tab-unit-information/DaycareAclAdditionModal.tsx:89]: Cannot reproduce :thinking:

**Review by reynders — APPROVED** [review]

**Pull request merged**